dashboard/config/linux/bits/base.yml

```diff
@@ -105,7 +105,7 @@ config:
  # This config does not add any debug checks (only debug output).
  - DEBUG_KOBJECT: n
 
- - DEBUG_INFO
+ - DEBUG_INFO: [-v5.18]
```

Comment:
We absolutely need DEBUG_INFO for symbolization. What have happened to it?

Reply:
https://lore.kernel.org/lkml/20220308161058.zuuze2jvhbiq53xy@maple.lan/T/